src/App.js

```diff
@@ -1,14 +1,36 @@
 import { Routes, Route } from "react-router-dom";
+import { useId, useState } from "react";
 
 import Layout from "./components/Layout";
 import ErrorPage from "./pages/ErrorPage";
 import Home from "./pages/Home";
 import Login from "./pages/Login";
 import New from "./pages/New";
 import Customer from "./pages/Customer"
-import Order from "./pages/Order";
+import Order from "./pages/NewOrder";
+import { toLocalStorage, fromLocalStorage } from "./utils/localStorage";
+
+
 
 function App() {
+  const [orderList, setOrderList] = useState(fromLocalStorage("orderList"));
+  const user = localStorage.getItem(`user`);
```

Comment:
Stay consistent in your usage of localStorage. If you're using `fromLocalStorage` then don't mix it with directly accessing `localStorage.getItem`.